src/chrome/content/rules/Mozilla.com.tw.xml

```diff
@@ -1,13 +1,19 @@
 <!--
 	For other Mozilla coverage, see Mozilla.xml.
-
+	
+	Invalid certificate:
```

Comment:
Yes, the comment is just not accurate. They have a valid cert, but serve an incomplete cert chain.